content/en/case-studies/blackhole-image.md

```diff
@@ -60,75 +64,71 @@ enough to read a newspaper in New York from a sidewalk café in Paris!
-imaging algorithms form the core of this task as through imaging, scientists
-could calculate the shadow of the black hole which forms the crux of several
-other calculations related to event horizon and nearby objects. One of the key
-algorithms used in imaging was developed by Katie Bouman – Continuous
-High-resolution Image Reconstruction using Patch priors, or ‘CHIRP’. It can
-parse the cumulative telescope data gathered by the Event Horizon Telescope
-project.For imaging tasks, researchers banked on Python to run the datasets on
-these algorithms, arraying and plotting data for meaningful insights.
-
-Besides NumPy, there were other packages such as
-[SciPy](https://www.scipy.org), [Pandas](https://pandas.io) and
-[Matplotlib](https://matplotlib.org) that helped in imaging and
-[data processing for imaging the black hole](https://iopscience.iop.org/article/10.3847/2041-8213/ab0c57).
-The standard astronomical file formats and time/coordinate transformations
-were handled by [Astropy](https://www.astropy.org) while Matplotlib was used
-in visualizing data throughout the analysis pipeline, including the generation
-of the final image of the black hole.
-
-Andrew Chael and the ehtim project team came up with
-[eht-imaging](https://github.com/achael/eht-imaging) Python modules for
-simulating and manipulating VLBI data and producing images with regularized
-maximum likelihood methods. NumPy is at the core of array data processing used
-in this software package named ehtim as indicated by the partial software
+For example, the [`eht-imaging`][ehtim] Python package provides tools for
+simulating and performing image reconstruction on VLBI data. 
+NumPy is at the core of array data processing used
+in this package as illustrated by the partial software
 dependency chart below.
 
 {{< figure src="/images/content_images/cs/ehtim_numpy.png" class="fig-center" alt="ehtim dependency map highlighting numpy" caption="**Software dependency chart of ehtim package highlighting NumPy**" >}}
 
-The challenge posed during reconstruction of an image using VLBI measurements
-is that there can be an infinite number of possible images that explain the
-data.  The ehtim software addresses this challenge by implementing algorithms
-that help find a set of most likely reasonable images that respects prior
-scientific assumptions while still satisfying the observed data.
+[ehtim]: https://github.com/achael/eht-imaging
+
+Besides NumPy, many other packages such as
+[SciPy](https://www.scipy.org) and [Pandas](https://pandas.io) were used in the
+data processing pipeline for imaging the black hole.
+The standard astronomical file formats and time/coordinate transformations
+were handled by [Astropy][astropy] while [Matplotlib][mpl] was used
+in visualizing data throughout the analysis pipeline, including the generation
+of the final image of the black hole.
 
+[astropy]: https://www.astropy.org/
+[mpl]: https://matplotlib.org/
```

Comment:
maybe make these normal (`[]()` markdown links?

Reply:
and the `ehtim` one above as well?

Reply:
oh, they were normal links before - I think that'd be more robust and just as easy

Reply:
Dealer's choice - I often use the markdown [name][key] link style as it makes the source text easier to read (especially when the links themselves are very long). I don't mind either way though.

Reply:
> I think that'd be more robust and just as easy

One last thought: both syntaxes are [commonmark compliant](https://commonmark.org/help/), so robustness shouldn't be an issue - more of a personal preference

Reply:
okay never mind then. I have actually never seen this used, but if it's compliant should be okay

Reply:
I'll try to take a look at these today. Thanks @rossbar and @rgommers.